.dprint.json

```diff
@@ -0,0 +1,44 @@
+{
+  "incremental": true,
+  "typescript": {},
+  "json": {},
+  "markdown": {
+    "lineWidth": 100
+  },
+  "toml": {},
+  "includes": [
+    "docs/**/*.{ts,js,cjs,mjs,json}",
+    "**/*.{md,toml}"
```

Comment:
Do you think there'd be value in adding `**/*.json` to catch `.dprint.json` file?

Reply:
Hmm, good idea. I think I had it in the previous version, but some stuff appears to have gotten lost in the rebase.

Reply:
@matchai Fixed, and I think I've also nuked some workflow regressions which snuck in.